src/Http/routes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::group(['prefix' => 'api/exponent/devices', 'middleware' => ['auth:api', 'bindings']], function () {
+Route::group(['prefix' => 'api/exponent/devices', "middleware" => config("exponent-push-notifications")["middleware"]], function () {
```

Comment:
Can we register the middleware in the container under a unique key for the package? eg: `expo.middleware` and reference it instead of accessing the config from the routes?

Reply:
Gave it a shot, check it out and see if I got the idea.